src/components/game.js

```diff
@@ -36,21 +36,152 @@ class Game extends React.Component {
     const board = [];
     for (let i = 0; i < data.site.siteMetadata.size; i += 1) {
       board[i] = new Array(data.site.siteMetadata.size);
+      for (let j = 0; j < data.site.siteMetadata.size; j += 1) {
+        board[i][j] = 0;
+      }
     }
     this.state = {
       board,
     };
+    this.handleKeyDown = this.handleKeyDown.bind(this);
   }
 
-  newTile = (board) => {
+  componentDidMount() {
```

Comment:
Piszę komentarz tutaj, bo nie jestem w stanie wyżej. 
szerokość 127.5px; można wyciągnąć do zmiennej bo wykorzystujesz w kilku miejscach

Reply:
Będe poprawiał to wszystko tak aby się skalowało. Ale to raczej pod koniec ;)